theseus/optimizer/nonlinear/tests/test_backwards.py

```diff
@@ -111,7 +111,7 @@ def fit_x(data_x_np):
         optimizer_kwargs={
             "track_best_solution": True,
             "verbose": False,
-            "backward_mode": th.BackwardMode.TRUNCATED,
+            "backward_mode": "TRUNCATED",
```

Comment:
nit: TRUNCATED -> truncated

Reply:
Oh, I was just adding a call with uppercase to add coverage for this case.